### PR TITLE
[3288] Added mission statement for Learning Team section.

### PR DESCRIPTION
**Pull request by kamizen**

### CODE REVIEW

File: source/team.html.erb [hunk 1]
```diff
@@ -144,7 +144,7 @@ responsive: true
     The Ember Learning Team
   </h2>
   <p class="lead text-center">
-    The Ember Learning Team is responsible for keeping the guides and API documentation accurate and up-to-date. They also manage the community outreach coordination and other initiatives that support learning Ember.
+    The Ember Learning Team empowers Ember users to learn, build and teach. They are responsible for keeping the guides and API documentation accurate and up-to-date. They also manage the community outreach coordination and other initiatives that support learning Ember.
```
Comment (MelSumner) [inline source/team.html.erb]: If it's not too much trouble, would you mind changing the text a little bit? I tried to streamline it so it was a little more efficient:

The mission of the Ember Learning Team is to empower Ember users to learn, build and teach. This team is responsible for keeping the guides and API documentation updated, and manages the initiatives that support learning Ember.

File: data/showcase.yml [hunk 2]
```diff
@@ -34,6 +34,18 @@
   features:
```
Comment (MelSumner) [inline data/showcase.yml]: Please remove this file from this PR. thank you! :)

**Comment by jessica-jordan** [comment]
@kamizen Thank you for sending this PR! Do you have time to take another look at it - after addressing the latest comments, this should be good to 🍏 already ✨

**Comment by jenweber** [comment]
This issue was covered by another PR at some point: https://github.com/emberjs/website/pull/3354/files back when all the other group descriptions were added too!

Thank you for your willingness to help!

**Pull request closed without merging**